charts/pxc-database/templates/_helpers.tpl

```diff
@@ -13,13 +13,13 @@ If release name contains chart name it will be used as a full name.
 */}}
 {{- define "pxc-database.fullname" -}}
 {{- if .Values.fullnameOverride -}}
-{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- .Values.fullnameOverride | trunc 21 | trimSuffix "-" -}}
```

Comment:
we truncate at 63 because of kube limitations on some names ( dns etc ). these are defaults created by `helm create` chart generator.   why the change to 21 ?

(same question for the trunc 3 etc )

Reply:
`status:
  conditions:
  - lastTransitionTime: "2019-07-25T08:55:04Z"
    message: 'wrong PXC options: cluster (pxc) name too long, must be no more than
      22 characters'
    reason: ErrorReconcile
    status: "True"
    type: Error
  message:
  - 'Error: wrong PXC options: cluster (pxc) name too long, must be no more than 22`

Reply:
discrepancy between trim length is just a typo

Reply:
ahhha!!